repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/content-cards.ftl

```diff
@@ -2,40 +2,43 @@
 <#import "./link.ftl" as hlink>
 
 <#macro contentCards contentCards size="third" relatedContent=false>
-    <#if contentCards.header?has_content>
-        <div class="nhsuk-u-reading-width">
-            <h2<#if relatedContent> data-anchorlinksignore="true"</#if>>${contentCards.header}</h2>
-        </div>
-    </#if>
-    <#if contentCards.cardGroupSummary?has_content>
-        <p>${contentCards.cardGroupSummary?replace('\n', '<br>')}</p>
-    </#if>
-    <ul class="nhsuk-grid-row nhsuk-card-group">
-        <#list contentCards.cards as contentCard>
-            <#if contentCard.header.document?? || contentCard.header.url?has_content>
-                <#assign clickableCard=true/>
-            <#else>
-                <#assign clickableCard=false/>
-            </#if>
+    <#if contentCards.cards??>
```

Comment:
do you want to update to <#if contentCards?? && contentCards.cards??> ?

Reply:
Pushed `contentCards?has_content`, will it capture both?